inference/interact/gui.py

```diff
@@ -21,13 +21,14 @@
 
 import numpy as np
 import torch
+from torch import mps
 
-from PyQt5.QtWidgets import (QWidget, QApplication, QComboBox, QCheckBox,
+from PyQt6.QtWidgets import (QWidget, QApplication, QComboBox, QCheckBox,
```

Comment:
Primarily switched to utilize the most up-to-date version of PyQt as a quality of life improvement, to ensure smooth support for future Python versions. An important note may be that PyQt6 only supports Python v3.6.1 and later, so even though it will support future Python versions, if it is needed to use older Python versions then I could revert this change.